pkg/repo/chartrepo_test.go

```diff
@@ -185,3 +189,82 @@ func verifyIndex(t *testing.T, actual *IndexFile) {
 		}
 	}
 }
+
+// StartLocalServerForTests Start the local helm server
+func StartLocalServerForTests(handler http.Handler) (*httptest.Server, error) {
```

Comment:
sure yes, actually earlier i wrote it thinking it would be used elsewhere too, but cannot use it anywhere else, but missed making it private, done now 👍